navyu_path_tracker_node:
  ros__parameters:
    map_frame: map
    base_frame: base_link
    update_frequency: 100.0
    limit_v_speed: 0.5
    limit_w_speed: 1.0
    yaw_tolerance: 0.07
    gain: 0.1 # to update look_ahead_distance from velocity
    look_ahead_const: 0.3
